topics/ui/controls/combo_box.md

```diff
@@ -23,79 +23,86 @@ comboBox.setEditable(true);
 
 ## When to use
 
-Use a combo box if:
+Use a combo-box if:
 
-* The user needs to select a value or enter a custom value.
+* The user needs to select a value or enter a custom value:
 
-  ![](combo_box_font_size.png){width=165}
+  ![A font size combo-box that allows selecting predefined values as well as entering a custom value](combobox_when_to_use_1.png){width=706}
 
 * The possible choices are objects or states.
```

Comment:
I'm not sure why we had this rule. What would be the alternative, actions? But I'm not sure anyone would want to put actions into the combo box. Let's remove it?

Reply:
I think the alternative here could be a dropdown list of options, but then you couldn't enter a custom value into it

Reply:
Agreed, let's remove it